Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN go mod download
 COPY . .
 RUN cd cmd/sso-auth && go build -mod=readonly -o /bin/sso-auth
 RUN cd cmd/sso-proxy && go build -mod=readonly -o /bin/sso-proxy
+RUN cd cmd/sso-proxy/generate-request-signature && go build -mod=readonly -o /bin/sso-generate-request-signature
```

Comment:
Adding some documentation for the new binary would be helpful for others to figure out how to use the tool.

Reply:
For sure, I'll add some documentation for it 👍

Reply:
Added a little markdown file in `/docs` explaining usage of the tool.